lib/game.ex

```diff
@@ -2,14 +2,38 @@ defmodule TicTacToe.Game do
   alias TicTacToe.{Board, HumanPlayer, ComputerPlayer, Validator, CliDisplay, Messager}
 
   def play_tic_tac_toe do
+    start_new_game(%{})
+  end
+
+  def start_new_game(player_scores) do
     welcome_players
     {player_1, player_2} = decide_game_type
-    start_new_game({player_1, player_2})
+    current_board = Board.create_new_board
+    player_scores = add_player_scores(player_scores, player_1, player_2)
+    take_turn(current_board, {player_1, player_2}, player_scores)
```

Comment:
`current_board`, `player_1`, `player_2`, and `player_scores` feel like a [data clump](http://www.martinfowler.com/bliki/DataClump.html)

Might be a good experiment to group these into a `Game` struct.